### PR TITLE
Document Controls, a closure and a FP from THM

### PR DESCRIPTION
**Pull request by GiudGiud**
refs #19737

This will make https://mooseframework.inl.gov/source/index.html#! look better. Less mixing of incompatible things

### CODE REVIEW

File: modules/thermal_hydraulics/doc/content/source/controllogic/PIDControl.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# PIDControl
+
+!syntax description /Controls/PIDControl
+
+The reference or target value is set by the [!param](/Controls/PIDControl/set_point) parameter control data.
+The value of the `output` data is set by:
+
+!equation
+\text{error} = \text{set point} - \text{value} \\
+\text{output} = K_p * \text{error} + \text{error integral} + K_d * \dfrac{\text{error} - \text{error}_{old}}{dt};
```
Comment (joshuahansel) [inline modules/thermal_hydraulics/doc/content/source/controllogic/PIDControl.md:10]: Need to mention K_i as well.

File: modules/thermal_hydraulics/doc/content/source/controllogic/CopyPostprocessorValueControl.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# CopyPostprocessorValueControl
+
+!syntax description /Controls/CopyPostprocessorValueControl
+
+This is useful for the numerous `ControlLogic` objects that rely on control data. Using this
```
Comment (joshuahansel) [inline modules/thermal_hydraulics/doc/content/source/controllogic/CopyPostprocessorValueControl.md:5]:
```suggestion
This is useful for the numerous [ControlLogic](ControlLogic/index.md) objects that rely on control data. Using this
```

**Review by joshuahansel — CHANGES_REQUESTED** [review]

**Comment by joshuahansel** [comment]
Note that I can help with addressing some of my comments, but it wouldn't be for another week or so.

File: modules/thermal_hydraulics/doc/content/source/controllogic/TerminateControl.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# TerminateControl
+
+!syntax description /Controls/TerminateControl
+
+The boolean that determines when to terminate is a `ControlData`. This is only used in the
```
[the remark this answers is not in the document]
Comment (GiudGiud) [reply modules/thermal_hydraulics/doc/content/source/controllogic/TerminateControl.md]: done.
I've added a bunch of links, see latest commit

File: modules/thermal_hydraulics/doc/content/source/controllogic/PIDControl.md (hunk 1 above)
Comment (GiudGiud) [reply modules/thermal_hydraulics/doc/content/source/controllogic/PIDControl.md:11]: done

**Comment by GiudGiud** [comment]
BISON doco patch is created and approved.

**Review by joshuahansel — APPROVED** [review]

**Pull request merged**